### DISCUSSION TITLE
Suggesting a change of Disks in Accessories and the Administration menu groups

### DISCUSSION THREAD

**Discussion by ineuw** [linuxmint/discussions#666] [Ideas]
'Disks' is located in 'Accessories', but requires root permission for some of it's options. On the other hand, 'Disk Analysis' has no such requirement. Logic dictates to, either reverse their grouping, or move 'Disks' to Administration as well.

**Comment by Jopp-gh** [comment]
You can create your own groupings, just right click on `menu@cinnamon. org` and then on "edit menu".

**Reply by ineuw** [reply]
It is not a question. This is a suggestion to group the menu apps that require admin rights. The Administration is the logical choice, and vice versa. This was a question was asked by users whom I am guiding in the use of LMC. This is my volunteer work for elderly computer users who have Windows experience through their past jobs.

Questions will be asked in the Forum.
